scripts/cleanup.js

```diff
@@ -125,7 +125,7 @@ async function cleanup() {
         console.log("DONE");
     }
     catch (error) {
```

Comment:
Oh, this is not runtime, there are maintenance scripts. Please revert all files in /scripts/ folder